src/style.css

```
@@ -1 +1,16 @@
 /* styles go here */
+.form-field {
+  display: block;
+}
+
+.form-field.inline {
+  display: inline-block;
+}
+
+fieldset .form-field:not(:last-child) {
```

Don't increase specifisity when it's unnecessary
```suggestion
.form-field:not(:last-child) {
```